src/main/config/config.c

```diff
@@ -203,38 +203,23 @@ void resetPidProfile(pidProfile_t *pidProfile)
 
     pidProfile->yaw_p_limit = YAW_P_LIMIT_MAX;
     pidProfile->dterm_average_count = 4;
-    pidProfile->dterm_lpf_hz = 0;    // filtering ON by default
+    pidProfile->dterm_lpf_hz = 70;    // filtering ON by default
```

Comment:
70/70 worked incredibly well in RC8. I'm not sure how the dterm averaging will change that.